nexia/const.py

```diff
@@ -58,7 +58,7 @@
     AIR_CLEANER_MODE_ALLERGY,
 ]
 
-HUMIDITY_MIN = 0.35
+HUMIDITY_MIN = 0.10
 HUMIDITY_MAX = 0.65
 
```

Comment:
Looks like this value isn't constant between systems

![AE62A76B-A979-4787-B137-1C4349117799_4_5005_c](https://github.com/user-attachments/assets/86584c90-d0e3-483c-812d-7b692cf2b7da)